frontend/src/scenes/notebooks/Notebook/notebookLogic.ts

```diff
@@ -82,10 +89,7 @@ export const notebookLogic = kea<notebookLogicType>([
                         throw new Error('Notebook not found')
                     }
 
-                    if (!values.notebook) {
```

Comment:
We only load notebooks when the component mounts so I thought it was ok to remove this check as it was preventing the refreshed content from being populated into the editor